website/content/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways.mdx

```diff
@@ -42,6 +41,13 @@ gateways](/docs/connect/gateways/mesh-gateway/service-to-service-traffic-datacen
 
 [![WAN federation with mesh gateways](/img/wan-federation-connectivity-mesh-gateways.png)](/img/wan-federation-connectivity-mesh-gateways.png)
 
+## Requirements
+
+- Consul 1.8.0+
+- All Consul datacenters must be connected to all other datacenters
+   - Other topologies, such as "hub-and-spoke" enabled by Consul Enterprise network areas,
+     are not supported
```

Comment:
```suggestion
- Consul 1.8.0+
- All Consul datacenters must be connected to all other datacenters. Other topologies, such  as _hub-and-spoke_  enabled by Consul Enterprise network areas, are not supported.
```
We had a contentious discussion amongst the TWs and one of the hot-button issues was when to use quotation marks and when to use italics. We will be updating the guidance to something like:

* Use quotation marks when required as a part of the code syntax and when referring to titles of books, articles, and other works. 
* Do not use quotation marks to introduce terms or to add emphasis. Use italics instead. 

Sorry for the back and forth.

I think the second requirement might work better as a single bullet because it's a qualification/clarification of the initial statement, rather than a sub requirement.